src/Arcus.Messaging.Pumps.ServiceBus/AzureServiceBusMessagePumpOptions.cs

```diff
@@ -34,9 +34,28 @@ public class AzureServiceBusMessagePumpOptions
         /// <remarks>When turned off, clients have to explicitly mark the messages as completed</remarks>
         public bool AutoComplete { get; set; }
 
+        /// <summary>
+        /// Gets or sets the unique identifier for this background job to distinguish this job instance in a multi-instance deployment.
+        /// </summary>
+        public string JobId { get; set; }
+
+        /// <summary>
+        /// Gets or sets the value indicating whether or not a new Azure Service Bus Topic subscription has to be created when the <see cref="AzureServiceBusMessagePump{TMessage}"/> starts.
+        /// The subscription will be deleted afterwards when the message pump stops.
+        /// REMARK: requires at least 'Manage' permissions on the Azure Service Bus Topic to create and delete these subscriptions.
+        /// </summary>
+        /// <remarks>
+        ///     Requires at least 'Manage' permissions on the Azure Service Bus Topic to create and delete these subscriptions.
```

Comment:
Manage is the highest permission
```suggestion
        ///     Requires 'Manage' permissions on the Azure Service Bus Topic to create and delete these subscriptions.
```